app/views/reports/regions/_search.html.erb

```diff
@@ -10,7 +10,7 @@
                          id: :search_query,
                          class: "typeahead-input form-control",
                          hide_label: true,
-                         placeholder: "Find regions",
+                         placeholder: "Search for a district, block, or facility",
```

Comment:
@rsanheim can we use "Search for a region or facility name" as placeholder text? Users that only have access to blocks or facilities won't be able to search by district. Using "regions" instead of specifying regions allows us to use a single placeholder text for all dashboard roles.

Reply:
Sure 👍